task1/README.md

```diff
@@ -0,0 +1,81 @@
+# Socket Chat
```

Comment:
add a description of params that you can pass when execute client.py or server.py